geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -69,18 +71,32 @@ <h3>{{ image_info.shelfmark }} {{ image_info.label }}</h3>
                     </div>
                 </div>
                 <div class="transcription-panel">
-                    {% if forloop.first %}
-                        <h3>Transcription</h3>
-                        <span data-transcription-target="editionFullLabel" class="current-transcription">{{ document.digital_editions.0.display|safe }}</span>
+                    {% if image_info.not_in_document %}
+                        {# if this image isn't actually in this document, link to related documents #}
+                        <ul class="excluded-related">
+                            {% for related_document in document.related_documents %}
+                                <li>
+                                    <a href="{% url 'corpus:document' related_document.pgpid %}">
+                                        <span class="doctype">{{ related_document.type }}</span>
+                                        <span class="shelfmark">{{ related_document.shelfmark|shelfmark_wrap }}</span>
```

Comment:
I see what you mean about how we're duplicating this, but I don't think a template snippet makes sense. Do we need the spans here? I guess we probably do need the shelfmark wrapping for linking to multishelfmark docs.

If we're concerned about the reuse and consistency, what about a method on `Document` like `display_title` or `formatted_title` that would be parallel to `title` but adds the spans/wrapping?

Reply:
Oh, but that doesn't help us with the solr results, which isn't actually a `Document` instance.  We could implement it on our solr result class, maybe, but again, not sure how much that helps.

This is still small enough I'm not sure I want the opacity of a template snippet.